scripts/app.js

```diff
@@ -584,6 +598,15 @@
 				utilities.loader.start();
 				utilities.api(listIntervalsForDate).then(this.handlePayPeriodRequest);
 			},
+			goToPreviousPayPeriod: function () {
+				let dates = this.getPayPeriodRange(this.$_currentPayPeriod);
+				this.getPayPeriod(dates[0].subtract(1, "days"));
+			},
+			goToNextPayPeriod: function () {
+				let dates = this.getPayPeriodRange(this.$_currentPayPeriod);
+				this.getPayPeriod(dates[1].add(11, "days"));
```

Comment:
Is 11 just an arbitrary number to put the date into the next pay period?

Reply:
Looking at this again, I have no idea. TBF I was buzzed up when I did this. ill get back to you on this

Reply:
@jomogalla I actually think it was meant to be a 1. dates[1] returns the endDate of a payPeriod. Incrementing it by 1 should give you the start of the next pay period.